scripts/servermanager.sh

```diff
@@ -0,0 +1,167 @@
+#!/bin/bash
+# shellcheck disable=SC1091
+
+# Stop on errors, comment in, if needed
+#set -e
+
+### Includes
+source /includes/config.sh
+source /includes/cron.sh
+source /includes/rcon.sh
+source /includes/security.sh
+source /includes/webhook.sh
+source /includes/colors.sh
+
+steamcmd_dir="/home/steam/steamcmd"
+
+### Server Functions
+
+function start_server() {
+    # IF Bash extension used:
+    # https://stackoverflow.com/a/13864829
+    # https://pubs.opengroup.org/onlinepubs/9699919799/utilities/V3_chap02.html#tag_18_06_02
+
+    pp --success "\n>>> Starting the gameserver\n"
+    cd "$GAME_ROOT" || exit
+
+    setup_configs
+
+    START_OPTIONS=""
+    if [[ -n $COMMUNITY_SERVER ]] && [[ $COMMUNITY_SERVER == "true" ]]; then
+        pp --info "> Setting Community-Mode to enabled\n"
+        START_OPTIONS="$START_OPTIONS EpicApp=PalServer"
+    fi
+    if [[ -n $MULTITHREAD_ENABLED ]] && [[ $MULTITHREAD_ENABLED == "true" ]]; then
+        pp --info "> Setting Multi-Core-Enhancements to enabled\n"
+        START_OPTIONS="$START_OPTIONS -useperfthreads -NoAsyncLoadingThread -UseMultithreadForDS"
+    fi
+    if [[ -n $WEBHOOK_ENABLED ]] && [[ $WEBHOOK_ENABLED == "true" ]]; then
+        send_webhook_notification "$WEBHOOK_START_TITLE" "$WEBHOOK_START_DESCRIPTION" "$WEBHOOK_START_COLOR"
+    fi
+    ./PalServer.sh "$START_OPTIONS"
+}
+
+function stop_server() {
```

Comment:
This function is so complex and brings unneeded complexity. The old version did what it should do. I dont understand why this exists.